Folder 4/hi_everyone.js

```diff
@@ -0,0 +1 @@
+// Hi everyone now im adding in a javascript file in a branch
```

Comment:
Block: this line introduces a bug, if the variable comes in undefined it will cause a red toast